source/software/r_package_management.rst

```diff
@@ -1,3 +1,5 @@
+.. _R_package_management:
+
 R package management
```

Comment:
I think it would also be good, in this R package management page, to refer to your new page about installing R packages with devtools (in the hope that sphinx does not complain about circular dependencies). Before trying that though, it might be worth to reconsider whether it would be possible after all to include your new stuff into this R package management page (rather than having it in a separate page). At first sight it looks like this could be done.